homeassistant/components/lutron_caseta/device_trigger.py

```diff
@@ -0,0 +1,273 @@
+    "on": 2,
+    "stop": 3,
+    "off": 4,
+    "raise": 5,
+    "lower": 6,
+}
+PICO_3_BUTTON_RAISE_LOWER_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(PICO_3_BUTTON_RAISE_LOWER_BUTTON_TYPES),
+    }
+)
+
+PICO_4_BUTTON_BUTTON_TYPES = {
+    "button_1": 8,
+    "button_2": 9,
+    "button_3": 10,
+    "button_4": 11,
+}
+PICO_4_BUTTON_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(PICO_4_BUTTON_BUTTON_TYPES),
+    }
+)
+
+
+PICO_4_BUTTON_ZONE_BUTTON_TYPES = {
+    "on": 8,
+    "raise": 9,
+    "lower": 10,
+    "off": 11,
+}
+PICO_4_BUTTON_ZONE_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(PICO_4_BUTTON_ZONE_BUTTON_TYPES),
+    }
+)
+
+
+PICO_4_BUTTON_SCENE_BUTTON_TYPES = {
+    "button_1": 8,
+    "button_2": 9,
+    "button_3": 10,
+    "off": 11,
+}
+PICO_4_BUTTON_SCENE_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(PICO_4_BUTTON_SCENE_BUTTON_TYPES),
+    }
+)
+
+
+PICO_4_BUTTON_2_GROUP_BUTTON_TYPES = {
+    "group_1_button_1": 8,
+    "group_1_button_2": 9,
+    "group_2_button_1": 10,
+    "group_2_button_2": 11,
+}
+PICO_4_BUTTON_2_GROUP_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(PICO_4_BUTTON_2_GROUP_BUTTON_TYPES),
+    }
+)
+
+FOUR_GROUP_REMOTE_BUTTON_TYPES = {
+    "open_all": 2,
+    "stop_all": 3,
+    "close_all": 4,
+    "raise_all": 5,
+    "lower_all": 6,
+    "open_1": 10,
+    "stop_1": 11,
+    "close_1": 12,
+    "raise_1": 13,
+    "lower_1": 14,
+    "open_2": 18,
+    "stop_2": 19,
+    "close_2": 20,
+    "raise_2": 21,
+    "lower_2": 22,
+    "open_3": 26,
+    "stop_3": 27,
+    "close_3": 28,
+    "raise_3": 29,
+    "lower_3": 30,
+    "open_4": 34,
+    "stop_4": 35,
+    "close_4": 36,
+    "raise_4": 37,
+    "lower_4": 38,
+}
+FOUR_GROUP_REMOTE_TRIGGER_SCHEMA = LUTRON_BUTTON_TRIGGER_SCHEMA.extend(
+    {
+        vol.Required(CONF_SUBTYPE): vol.In(FOUR_GROUP_REMOTE_BUTTON_TYPES),
+    }
+)
+
+DEVICE_TYPE_SCHEMA_MAP = {
+    #    "Pico1Button": HELP WANTED - DO NOT HAVE TO TEST - NEED DEVICE MAPPINGS,
+    "Pico2Button": PICO_2_BUTTON_TRIGGER_SCHEMA,
+    #    "Pico2ButtonRaiseLower": HELP WANTED - DO NOT HAVE TO TEST - NEED DEVICE MAPPINGS,
+    #    "Pico3Button": HELP WANTED - DO NOT HAVE TO TEST - NEED DEVICE MAPPINGS,
+    "Pico3ButtonRaiseLower": PICO_3_BUTTON_RAISE_LOWER_TRIGGER_SCHEMA,
+    "Pico4Button": PICO_4_BUTTON_TRIGGER_SCHEMA,
+    "Pico4ButtonScene": PICO_4_BUTTON_SCENE_TRIGGER_SCHEMA,
+    "Pico4ButtonZone": PICO_4_BUTTON_ZONE_TRIGGER_SCHEMA,
+    "Pico4Button2Group": PICO_4_BUTTON_2_GROUP_TRIGGER_SCHEMA,
+    "FourGroupRemote": FOUR_GROUP_REMOTE_TRIGGER_SCHEMA,
+}
+
+DEVICE_TYPE_SUBTYPE_MAP = {
+    #    "Pico1Button": HELP WANTED - DO NOT HAVE TO TEST - NEED DEVICE MAPPINGS,
```

Comment:
I'm not sure these were ever produced as I can't find them in the lutron product catalog or on ebay.

Reply:
I don't ever recall having seen one of these before (and I worked there until last September).

Reply:
I'll remove this line